playbooks/common/openshift-etcd/scaleup.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Configure etcd
+  hosts: oo_new_etcd_to_config
+  serial: 1
+  any_errors_fatal: true
+  vars:
+    etcd_ca_host: "{{ groups.oo_etcd_to_config.0 }}"
+    etcd_peer_url_scheme: https
+    etcd_peer_port: 2380
+  pre_tasks:
+  - name: Add new etcd members to cluster
+    command: >
+      /usr/bin/etcdctl  --cert-file /etc/etcd/peer.crt
```

Comment:
We could also add the `etcd_common` role here prior to tasks and rely on these vars https://github.com/openshift/openshift-ansible/blob/master/roles/etcd_common/defaults/main.yml